COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -876,7 +884,7 @@ namespace COLLADASaxFWL
     }
 
     //------------------------------
-    void MeshLoader::initializeColorsOffset ()
+    bool MeshLoader::initializeColorsOffset ()
```

Comment:
nevertheless the function internally calls the application error handler. But when the application error handler returns "abort parsing" then this function should not silently ignore this request, but give this information back to the caller.

Imho the correct fix would be to avoid calling the application error handler from within initializeColorsOffset () but i can not judge if that is right or wrong, so i guess its better to make another patch for fixing that part? 

as a sidenote, from my work with blender i learned to make many small commits rather than collections of loosely related fixes...